File: python/wieksza.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-



def main(args):
    a = int(input("Podaj liczbe:"))
    b = int(input("Podaj liczbe:"))
    if a > b:
        print("a > b")
    elif b > a:
        print("b > a")
    else:
        print("a = b")
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: c++/wieksza.cpp
/*
 * wieksza.cpp
 */


#include <iostream>
using namespace std;

int main(int argc, char **argv)
{
	int a, b;
    cout << "Podaj liczbe a: ";
    cin >> a;
    cout << "podaj liczbe b: ";
    cin >> b;
    if (a > b) {
        cout << "a > b";
    } else if(b > a) { 
        cout << " a < b";
    } else {
        cout << " a = b";
        }
       
	return 0;
}

